app/handlers/analysis.py

```diff
@@ -18,4 +26,17 @@ class AnalysisHandler(tornado.web.RequestHandler):
     @apiError error_message The error message to display.
     """
     def post(self):
-        self.finish("Analysis")
+        identifier = self.get_body_argument("identifier")
+
+        ObjectTrackingHandler.handler(identifier)
+        SafetyAnalysisHandler.handler(identifier)
```

Comment:
What if the first handler fails but the second one succeeds? A check should be done in the middle, however unlikely the occurance may be.